.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+cache: yarn
+script:
+  - yarn --version
+  - yarn setup
+  - yarn check --integrity
+  - yarn lint
+  - yarn test
+  - yarn test:coverage
+before_install:
+  - yarn global add greenkeeper-lockfile@1
+before_script: greenkeeper-lockfile-update
+after_script: greenkeeper-lockfile-upload
```

Comment:
This should publish on new tag pushed, 
Something like this (the key will be different):
https://github.com/hmcts/look-and-feel/blob/master/.travis.yml#L9